### PR TITLE
Add cards from Scarlet Witch pack

### PR DESCRIPTION
**Pull request by lostie**
**Also:**

* Fix some issues with existing cards:
  * https://github.com/zzorba/marvelsdb/issues/82
  * https://github.com/zzorba/marvelsdb/issues/79

### CODE REVIEW

**Comment by lostie** [comment]
WIP, adding all remaining cards from the SW's pack

**Comment by lostie** [comment]
Asking @Kamalisk @wargfn @rdonnelly for review 👀

File: pack/scw.json [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+		"code": "15012",
+		"cost": 3,
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"name": "Crisis Averted",
+		"pack_code": "scw",
+		"position": 12,
+		"quantity": 3,
+		"resource_mental": 1,
+		"text": "<b>Hero Action</b> <i>(thwart)</i>: Remove 6 threat from the main scheme. If you paid for this card using a [mental] resource, this thwart ignores the crisis icon ([crisis]).",
```
Comment (lostie) [inline pack/scw.json:255]: I'm not sure if the `[crisis]` icon is supported as I could not find any usage of it in the code.

Comment (rdonnelly) [reply pack/scw.json:255]: It does not look like it is currently supported, but _should_ be a pretty straight-forward update in the db code. See here for the icon replacements that the app currently handles:

https://github.com/zzorba/marvelsdb/blob/7a94978e2c7ab267443bd5c9e45cb38c95d95681/src/AppBundle/Services/CardsData.php#L34

File: pack/scw.json [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"flavor": "\"Return what you've taken and I'll let you go. This time. But next time, you won't be so lucky. And I won't be so kind.\" - Moon Knight",
+		"name": "Swift Retribution",
+		"pack_code": "scw",
+		"position": 14,
+		"quantity": 3,
+		"resource_mental": 1,
+		"text": "<b>Hero Action</b> <i>(attack)</i>: The villain schemes. Deal 4 damage to the villain.",
+		"traits": "Thwart.",
```
Comment (rdonnelly) [inline pack/scw.json:287]: This trait should be "Attack."

```suggestion
		"traits": "Attack.",
```

File: pack/scw.json [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"flavor": "\"Back, you rabble!\" - Scarlet Witch",
+		"name": "Turn the Tide",
+		"pack_code": "scw",
+		"position": 15,
+		"quantity": 3,
+		"resource_energy": 1,
+		"text": "<b>Response</b> <i>(attack)</i>: After your hero thwarts and removes all threat from a scheme, deal 3 damage to an enemy.",
+		"type_code": "event"
```
Comment (rdonnelly) [inline pack/scw.json:302]: Needs an "Attack." trait.

```suggestion
		"text": "<b>Response</b> <i>(attack)</i>: After your hero thwarts and removes all threat from a scheme, deal 3 damage to an enemy.",
		"traits": "Attack."
		"type_code": "event"
```

File: pack/scw.json [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+		"flavor": "\"There is no defense against the Scarlet Witch!\"",
+		"illustrator": "Miguel Sepulveda",
+		"name": "Molecular Decay",
+		"pack_code": "scw",
+		"position": 5,
+		"quantity": 3,
+		"resource_energy": 1,
+		"set_code": "scw",
+		"set_position": 7,
+		"text": "<b>Hero Action</b> <i>attack</i>: Deal 5 damage to an enemy and discard the top 2 cards of the encounter deck. For each boost icon discarded this way, deal 1 additional damage to that enemy.",
```
Comment (rdonnelly) [inline pack/scw.json:122]: Just some missing parens:

```suggestion
		"text": "<b>Hero Action</b> <i>(attack)</i>: Deal 5 damage to an enemy and discard the top 2 cards of the encounter deck. For each boost icon discarded this way, deal 1 additional damage to that enemy.",
```

File: pack/scw_encounter.json [hunk 5]
```diff
@@ -0,0 +1,88 @@
+[
+	{
+		"boost": 3,
+		"code": "14023",
```
Comment (rdonnelly) [inline pack/scw_encounter.json:4]: All the codes in `saw_encounter.json` need to be "15XXX" instead of "14XXX".

**Review by rdonnelly — CHANGES_REQUESTED** [review]
Hey, very nice! Just a few small things I could comparing to Hall of Heroes:

https://hallofheroeslcg.com/scarlet-witch/

Thanks for doing this!

**Comment by lostie** [comment]
> Hey, very nice! Just a few small things I could comparing to Hall of Heroes:
> 
> https://hallofheroeslcg.com/scarlet-witch/
> 
> Thanks for doing this!

Thanks so much for taking the time to review it! I've addressed all the things you pointed out 🙇‍♂️

**Pull request merged**